encryption-at-rest.md

```diff
@@ -107,6 +107,19 @@ region = "us-west-2"
 
 在调试方面，可使用 `tikv-ctl` 命令查看加密元数据（例如使用的加密方法和数据密钥列表）。该操作可能会暴露密钥，因此不推荐在生产环境中使用。详情参阅 [TiKV Control](/tikv-control.md#打印加密元数据)。
 
+### TiKV 版本间兼容性
```

Comment:
@yiwu-arbug The English title is `Compatibility with earlier TiKV versions`. Can we make the two versions literally as close as possible?

Reply:
updated the English version as well.

Reply:
OK